### PR TITLE
Fix rtop

### PR DESCRIPTION
**Pull request by chenglou**
All these meta files are making me hungry!

See notes here:
https://github.com/facebook/reason/pull/659/files#r73099166

This fixes it.

### CODE REVIEW

**Comment by jordwalke** [comment]
Thank you, @chenglou !

**Comment by chenglou** [comment]
**Don't merge this yet**! #675 should currently be failing CI but it's not. Once that one successfully fails, we can merge this fix.

**Comment by chenglou** [comment]
LOL talk about timing lol. Alright well, I'll see how I can fix #675.

File: src/rtop_init.ml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env ocaml
 #require "menhirLib";;
+#require "re.str";;
```
Comment (rickyvetter) [inline src/rtop_init.ml]: Gah! I didn't know this file existed. I kept just making the other change and not being able to fix it. Thank you :)

**Pull request merged**